src/network/networkd-routing-policy-rule.c

```diff
@@ -790,7 +790,7 @@ int link_request_static_routing_policy_rules(Link *link) {
 }
 
 int request_process_routing_policy_rule(Request *req) {
-        RoutingPolicyRule *ret = NULL;  /* avoid false maybe-uninitialized warning */
+        RoutingPolicyRule *ret = NULL;
```

Comment:
did you intend to remove the `= NULL` here too?

Reply:
I meant to, but gcc warns when that is done. I dropped that chunk now.